dvc/repo/stage.py

```diff
@@ -31,6 +31,9 @@ class StageInfo(NamedTuple):
     stage: "Stage"
     filter_info: Optional[PathInfo] = None
 
+    def commit(self) -> None:
+        self.stage.commit(filter_info=self.filter_info)
+
```

Comment:
Could you explain why we need this and can't just pass filter_info to stage.commit() in dvc/repo/commit.py?

Reply:
Done - changed solution